.github/workflows/snyk.yml

```diff
@@ -1,14 +1,22 @@
-# This action runs every day at 6 AM and on every push
+# This action runs every day at 6 AM, on every push not from dependabot, and every pull_request from dependabot
 # If the branch it's running on is main then it will run snyk monitor (reports vulnerabilities to snyk.io)
 # Otherwise it will run snyk test
 name: DCR Snyk
 
 on:
     schedule:
         - cron: '0 6 * * *'
-    push:
+    push: 
```

Comment:
Totally unimportant: I think there's a trailing whitespace on this line?

Reply:
```suggestion
    push:
```